packages/vela/src/utils.ts

```diff
@@ -72,6 +72,35 @@ export function getOrderTypePacked(
   }
 }
 
+export function getOrderType(
+  orderType: OrderType | undefined,
+): { _orderType: undefined } | { $or: { _orderType: number }[] } | undefined {
```

Comment:
Could clean this up a bit with defined types - this reads a little gnarly but I'm not strongly convicted on that cleanup.